tests/consensus_test.rs

```diff
@@ -770,3 +784,93 @@ async fn test_read_node() {
         wait_for_read_node_blocks(read_node, 1).await;
     }
 }
+
+#[tokio::test]
+#[serial]
+async fn test_cross_shard_interactions() {
+    let env_filter = EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("warn"));
+    let _ = tracing_subscriber::fmt()
+        .with_env_filter(env_filter)
+        .try_init();
```

Comment:
Delete? If we need logs for debugging there's an `test_helper::enable_logging`

Reply:
Ah! Sorry copied that from another test, fixed!